packages/wifi-signal-strength/default.nix

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenvNoCC,
+  pkgs,
+  lib,
+  ...
+}: let
+  wifiSignalStrength =
+    pkgs.writeShellScript
+    "wifi-signal-strength"
+    ''
+      export DBUS_SESSION_BUS_ADDRESS=unix:path=/tmp/ssh_session_dbus.sock
+      export DBUS_SYSTEM_BUS_ADDRESS=unix:path=/tmp/ssh_system_dbus.sock
+      ${pkgs.openssh}/bin/ssh -M -S /tmp/nmcli_socket \
+          -f -N -q ghaf@192.168.100.1 \
+          -i /run/waypipe-ssh/id_ed25519 \
+          -o StrictHostKeyChecking=no \
+          -o StreamLocalBindUnlink=yes \
+          -o ExitOnForwardFailure=yes \
+          -L /tmp/ssh_session_dbus.sock:/run/user/1000/bus \
+          -L /tmp/ssh_system_dbus.sock:/run/dbus/system_bus_socket
+      signal0=󰤟
+      signal1=󰤢
+      signal2=󰤥
+      signal3=󰤨
+      no_signal=󰤭
+      # Get IP address of netvm
+      address=$(${pkgs.networkmanager}/bin/nmcli device show wlp0s5f0 | ${pkgs.gawk}/bin/awk '{ if ($1=="IP4.ADDRESS[1]:") {print $2}}')
+      # Get signal strength and ssid
+      connection=($(${pkgs.networkmanager}/bin/nmcli -f IN-USE,SIGNAL,SSID dev wifi | ${pkgs.gawk}/bin/awk '/^\*/{if (NR!=1) {print $2; print $3}}'))
+      connection[0]=$(if [ -z ''${connection[0]} ]; then echo "-1"; else echo ''${connection[0]}; fi)
+      # Set the icon of signal level
+      signal_level=$(if [ ''${connection[0]} -gt 80 ]; then echo $signal3; elif [ ''${connection[0]} -gt 60 ]; then echo $signal2; elif [ ''${connection[0]} -gt 30 ]; then echo $signal1; elif [ ''${connection[0]} -gt 0 ]; then echo signal0; else echo $no_signal; fi)
+      tooltip=$(if [ -z ''${connection[1]} ]; then echo "No connection"; else echo ''${connection[1]} ''${connection[0]}%; fi)
+      text=$(if [ -z $address ]; then echo $signal_level; else echo $address $signal_level; fi)
+      # Return as json format for waybar
+      echo "{\"percentage\":\""''${connection[0]}"\", \"text\":\""$text"\", \"tooltip\":\""$tooltip"\", \"class\":\"1\"}"
+      # Use the control socket to close the ssh tunnel.
+      ${pkgs.openssh}/bin/ssh -q -S /tmp/nmcli_socket -O exit ghaf@192.168.100.1
```

Comment:
You can use the variable here as well - https://github.com/tiiuae/ghaf/pull/386/commits/7f4b76d3db68579283410d47443bc73da4730629#r1447358171